.circleci/config.yml

```diff
@@ -3627,135 +3635,80 @@ workflows:
   version: 2
   build:
     jobs:
-      - pytorch_linux_trusty_py2_7_9_build
```

Comment:
for the removal of all of these lines, and addition of smoketest lines, you'll also need to make a change to the config generator.

Reply:
I don't want to remove those lines. The second commit is a temporary thing just so I can run the tests.